pkg/api/mlflow/dao/models/app.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"encoding/json"
+
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type App struct {
+	ID         uuid.UUID `gorm:"type:uuid;primaryKey" json:"id"`
+	CreatedAt  time.Time `json:"created_at"`
+	UpdatedAt  time.Time `json:"updated_at"`
+	IsArchived bool      `json:"-"`
+	Type       string    `gorm:"not null" json:"type"`
+	State      AppState  `json:"state"`
+}
+
+type AppState map[string]any
```

Comment:
probably if we add something will be nice to add comments everywhere.

Reply:
will do

Reply:
removing file